build/bootstrap_mcs.sh

```diff
@@ -104,7 +112,7 @@ install_deps() {
       cppunit-devel cmake3 libxcrypt-devel xz-devel zlib-devel libzstd-devel glibc-devel"
 
     DEB_BUILD_DEPS="apt-get -y update && apt-get -y install build-essential automake libboost-all-dev \
-      bison cmake libncurses5-dev libaio-dev libsystemd-dev libpcre2-dev libperl-dev libssl-dev libxml2-dev \
+      bison cmake libncurses5-dev python3 libaio-dev libsystemd-dev libpcre2-dev libperl-dev libssl-dev libxml2-dev \
```

Comment:
I am missing why do we need python for building?

Reply:
I saw why we need it. Can't we get away with c++ binary instead of using python?

Reply:
I don't think you want me to rewrite a python script